scripts/scaffold.js

```diff
@@ -36,11 +36,26 @@ export const Default = () => (
 );
 `;
 
+const TESTS = `
+import React from "react";
+import { shallow } from "enzyme";
+
+import { ${componentName} } from "./${componentName}";
+
+describe("${componentName}", () => {
+  it("renders correctly", () => {
+    const component = shallow(<${componentName} />);
+    expect(component.text()).toEqual("Hello");
+  });
+});
+`;
```

Comment:
I think having the scaffold throw together an immediately running story and spec is going to go a long way in getting real coverage for stuff.

Reply:
ONE HUNDRED. AND A FAILING TEST AT THAT :D